coderd/rbac/policy.rego

```diff
@@ -148,6 +148,22 @@ user_allow(roles) := num {
     num := number(allow)
 }
 
+# Scope allow_list is a list of resource IDs explicitly allowed by the scope.
+# If the list is '*', then all resources are allowed.
+scope_allow_list {
+	"*" in input.subject.scope.allow_list
+}
+
+scope_allow_list {
+	# If the wildcard is listed in the allow_list, we do not care about the
+	# object.id. This line is included to prevent partial compilations from
+	# ever needing to include the object.id.
+	not "*" in input.subject.scope.allow_list
+	input.object.id in input.subject.scope.allow_list
+}
+
+
+
```

Comment:
nit: too many empty lines

Reply:
so right. So used to go fmt being automatic.